docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,25 +9,44 @@ ms.topic: conceptual
 
 # Select Assemblies Referenced By Projects
 
-Explicit assembly references allows a subset of assemblies to be used for IntelliSense and compiling, while all assemblies are available at run-time. `PackageReference` and `packages.config` work differently, and as a result package authors need to take care to create the package to be compatible with both project types.
+Assemblies are used in two different ways during a build. The first is for compile, which allows the package consumer's code to compile against APIs in the assembly, and for Intellisense to give suggestions. The second is runtime, where the assembly is copied to the `bin` directory and is used at runtime.
 
-> [!Note]
-> Explicit assembly references are related to .NET assemblies. It is not a method to distribute native assemblies that are P/Invoked by a managed assembly.
+## Recommendation: Package dependencies and include/exclude assets
 
-## `PackageReference` support
+Our recommendation is to have one package per assembly, in which case you can use `PrivateAssets="compile"` in your `PackageReference` or `ProjectReference`. This provides more flexibility and minimizes risk that two different packages will contain different versions of the same assembly, preventing NuGet from being able to manage versions.
 
-When a project uses a package with `PackageReference` and the package contains a `ref\<tfm>\` directory, NuGet will classify those assembles as compile-time assets, while the `lib\<tfm>\` assemblies are classified as runtime assets. Assemblies in `ref\<tfm>\` are not used at runtime. This means it is necessary for any assembly in `ref\<tfm>\` to have a matching assembly in either `lib\<tfm>\` or a relevant `runtime\` directory, otherwise runtime errors will likely occur. Since assemblies in `ref\<tfm>\` are not used at runtime, they may be [metadata-only assemblies](https://github.com/dotnet/roslyn/blob/main/docs/features/refout.md) to reduce package size.
+```xml
+<ItemGroup>
+  <ProjectReference Include="..\OtherProject\OtherProject.csproj" PrivateAssets="compile" />
```

Comment:
Given that you're overwriting PrivateAssets compile, the examples should probably include the defaults as well. 
Expect people to copy things like this.